subsys/net/ip/ipv4.c

```diff
@@ -317,7 +317,8 @@ enum net_verdict net_ipv4_input(struct net_pkt *pkt, bool is_loopback)
 	}
```

Comment:
Please change the commit subject to `net: ipv4: Allow IGMP packets with zero src address`
and also add `Fixes #69917` before the signed-off-by line so that the commit is linked to the issue.